README.md

```diff
@@ -5,44 +5,44 @@ Our open source community's Wiki focuses on answering your questions about learn
 - Curriculum
 - Other Programming Languages
 - Local Campsite Communities
-- Nonprofit Projects  
+- Nonprofit Projects
  ...and much more
```

Comment:
and much more in same line? Was it intended?
I think It was looking better in separate lines.